Dev/agg setup

ArunPrasad017 opened the pull request.

ArunPrasad017 committed:
Adding agg query to run

ArunPrasad017 committed:
Remove the app destroy step

ArunPrasad017 committed:
removing the unwanted utils function

ArunPrasad017 committed:
correcting the query

ArunPrasad017 committed:
adding log statements

ArunPrasad017 committed:
correcting sql statement

ArunPrasad017 committed:
temporarily run destroy

Stegallo commented on datasette/my_example.py:
```diff
@@ -12,8 +12,6 @@
-    print(f"Inside the docker file at - {timestamp}")
-    log.debug("This is a debugging message")
     db = sqlite_utils.Database("laliga1819.db")
     db["laliga1819"].insert_all(
```
we can test that the requests.get call is made using a mock of request and using patch in the test case.
patch is used to replace the actual request object with a fake version that we can control (and has no link to the external url).
resources about mock and patch https://realpython.com/python-mock-library/#patch

Stegallo commented on datasette/my_example.py:
```diff
@@ -22,9 +20,19 @@ def initialize_db(timestamp: datetime) -> datetime:
+        sqlFile = fd.read()
+    db = sqlite_utils.Database("laliga1819.db")
+    log.info("Executing SQL file on laliga db")
+    db.execute(sqlFile)
```
to test this function we can create a "test" version of the database and pass to the function via a patch.
In addition we can test exception handling

ArunPrasad017 committed:
Added
a) Try catch for db
b) Try catch for requests path
c) stopping deployment booleans
d) Remove datetime

ArunPrasad017 committed:
type requests added to requirements.txt

ArunPrasad017 committed:
add dependency for pylint

ArunPrasad017 committed:
add dependency for pylint

ArunPrasad017 committed:
Test for init db

ArunPrasad017 committed:
Test for read

ArunPrasad017 committed:
Adding tests for functionalities in source

ArunPrasad017 committed:
correcting as per the tox checls
